crime-evidence/src/main/java/uk/gov/justice/laa/crime/evidence/service/MaatCourtDataService.java

```diff
@@ -20,14 +18,14 @@ public class MaatCourtDataService {
     private final ServicesConfiguration configuration;
     private static final String RESPONSE_STRING = "Response from Court Data API: %s";
 
-    public Long getRepOrderCapitalByRepId(Integer repId) {
-
-        ResponseEntity<Void> response = maatAPIClient.head(
-                configuration.getMaatApi().getRepOrderEndpoints().getRepOrderCapitalUrl(),
-                Map.of(),
+    public Integer getCapitalAssetCount(Integer repId) {
```

Comment:
This could be an opportunity potentially to change the return type here to a primitive rather than reference type, if there is no need for the reference type upstream. Will leave this as a suggestion, it could also be an exercise in itself across the codebase as a separate piece of refactoring.